presentation/src/main/res/layout/text_question_step_fragment.xml

```diff
@@ -16,12 +17,28 @@
         android:id="@+id/question_input"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:layout_below="@id/question_header"/>
+        android:layout_below="@id/question_header">
+
+    </org.sagebionetworks.assessmentmodel.presentation.inputs.TextInputView>
 
-    <org.sagebionetworks.assessmentmodel.presentation.views.StepNavigationBar
-        android:id="@+id/nav_bar"
+    <androidx.constraintlayout.widget.ConstraintLayout
```

Comment:
Is this ConstraintLayout doing anything?

Reply:
Yeah it is constraining the bar to the bottom of the screen.